locations/spiders/meijer.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import ast
+from locations.items import GeojsonPointItem
+
+STATES = ['AL', 'AK', 'AZ', 'AR', 'CA', 'CO', 'CT', 'DC', 'DE', 'FL', 'GA',
+          'HI', 'ID', 'IL', 'IN', 'IA', 'KS', 'KY', 'LA', 'ME', 'MD',
+          'MA', 'MI', 'MN', 'MS', 'MO', 'MT', 'NE', 'NV', 'NH', 'NJ',
+          'NM', 'NY', 'NC', 'ND', 'OH', 'OK', 'OR', 'PA', 'RI', 'SC',
+          'SD', 'TN', 'TX', 'UT', 'VT', 'VA', 'WA', 'WV', 'WI', 'WY']
+
+class MeijerSpider(scrapy.Spider):
+    name = 'meijer'
+    allowed_domains = ['www.meijer.com']
+
+    def start_requests(self):
+        for state in STATES:
+            yield scrapy.Request(
+                'https://www.meijer.com/custserv/locate_store_by_state.cmd?form_state=locateStoreByStateForm&state={}'.format(state),
+                callback = self.parse
+            )
+    
+    def parse(self, response):
+        selector = scrapy.Selector(response)
+        stores = selector.css('div.records_inner>script::text').extract_first()
+        if stores:
+            for store in ast.literal_eval(stores.strip()[13:-1]):
+                address1 = store[6].split(',')
+                city = address1[0].strip()
+                address2 = address1[1].strip().split(' ')
+                state = address2[0]
+                postcode = address2[1]
+                properties = {
+                    'ref': store[0],
+                    'name': store[1],
+                    'phone': store[7],
+                    'opening_hours': self.hours(store[8]),
+                    'lat': store[37],
```

Comment:
If you take a look at the [latest results](https://s3.amazonaws.com/placescraper-results/results/meijer/2017-12-11-01-16-33/output.geojson) it looks like the lat and lon are getting incorrect values.

Reply:
Currently I can see there is no value for lat and lon in query output. As this point we can keep lat and lon empty?

Reply:
[['274','Grafton','1600 N. Port Washington Road','1600 N. Port Washington Road','','/assets/temp/1_marker.png','Grafton, WI 53024','(262) 204-2800','Open 24 hrs a day, 364 days a year.','(262) 204-2810','9:00AM - 9:00PM','9:00AM - 7:00PM','10:00AM - 6:00PM','10:00AM - 6:00PM','Grafton','WI','53024','true','0','false','false','false','(262) 204-2800','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','true','In-Store','false','(877) 363-4537','9:00AM - 9:00PM','9:00AM - 7:00PM','9:00AM - 7:00PM','9:00AM - 7:00PM','false','','false','false'],['275','Sussex','N51 W 24953 Lisbon Rd','N51 W 24953 Lisbon Rd','','/assets/temp/2_marker.png','Pewaukee, WI 53072','(262) 932-2500','Open 24 hrs a day, 364 days a year.','(262) 932-2510','9:00AM - 9:00PM','9:00AM - 7:00PM','10:00AM - 6:00PM','closed','Pewaukee','WI','53072','true','0','false','false','false','(262) 932-2500','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','true','In-Store','false','(877) 363-4537','12:00AM - 12:00AM','12:00AM - 12:00AM','12:00AM - 12:00AM','12:00AM - 12:00AM','false','','true','false'],['276','Waukesha','801 E Sunset Dr','801 E Sunset Dr','','/assets/temp/3_marker.png','Waukesha, WI 53189','(262) 899-6000','Open 24 hrs a day, 364 days a year.','(262) 899-6010','9:00AM - 9:00PM','9:00AM - 7:00PM','closed','closed','Waukesha','WI','53189','true','0','false','false','false','(262) 899-6000','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00PM - 8:00PM','true','In-Store','false','(877) 363-4537','12:00AM - 12:00AM','12:00AM - 12:00AM','12:00AM - 12:00AM','12:00AM - 12:00AM','false','','true','false'],['277','Wauwatosa','11111 W. Burleigh','11111 W. Burleigh','','/assets/temp/4_marker.png','Wauwatosa, WI 53222','(414) 290-0900','Open 6:00 am - 11:00 pm Daily','(414) 290-0910','9:00AM - 9:00PM','9:00AM - 7:00PM','10:00AM - 6:00PM','10:00AM - 6:00PM','Wauwatosa','WI','53222','true','0','false','false','false','(414) 290-0900','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','true','In-Store','false','(877) 363-4537','9:00AM - 9:00PM','9:00AM - 7:00PM','9:00AM - 7:00PM','9:00AM - 7:00PM','false','','false','false'],['283','Oak Creek','171 W. Town Square Way','171 W. Town Square Way','','/assets/temp/5_marker.png','Oak Creek, WI 53154','(414) 501-1700','Open 24 hrs a day, 364 days a year.','(414) 501-1710','9:00AM - 9:00PM','9:00AM - 7:00PM','10:00AM - 6:00PM','10:00AM - 6:00PM','Oak Creek','WI','53154','true','0','false','false','false','(414) 501-1700','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','true','In-Store','false','(877) 363-4537','9:00AM - 9:00PM','9:00AM - 7:00PM','9:00AM - 7:00PM','9:00AM - 7:00PM','false','','false','false'],['284','Kenosha','7701 Green Bay Rd','7701 Green Bay Rd','','/assets/temp/6_marker.png','Kenosha, WI 53142','(262) 612-1600','Open 24 hrs a day, 364 days a year.','(262) 612-1610','9:00AM - 9:00PM','9:00AM - 7:00PM','10:00AM - 6:00PM','10:00AM - 6:00PM','Kenosha','WI','53142','true','0','false','false','false','(262) 612-1600','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','true','In-Store','false','(877) 363-4537','9:00AM - 9:00PM','9:00AM - 7:00PM','9:00AM - 7:00PM','9:00AM - 7:00PM','false','','false','false'],['292','Greenfield','5800 W. Layton Ave.','5800 W. Layton Ave.','','/assets/temp/7_marker.png','Greenfield, WI 53220','(414) 304-2000','Open 24 hrs a day, 364 days a year.','(414) 304-2010','9:00AM - 9:00PM','9:00AM - 7:00PM','10:00AM - 6:00PM','closed','Greenfield','WI','53220','true','0','false','true','false','(414) 304-2047','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','true','In-Store','false','(877) 363-4537','12:00AM - 12:00AM','12:00AM - 12:00AM','12:00AM - 12:00AM','12:00AM - 12:00AM','false','','true','false'],['298','Howard','2015 Shawano Ave','2015 Shawano Ave','','/assets/temp/8_marker.png','Green Bay, WI 54303','(920) 496-8100','Open 24 hrs a day, 364 days a year.','(920) 496-8110','9:00AM - 9:00PM','9:00AM - 7:00PM','10:00AM - 6:00PM','closed','Green Bay','WI','54303','true','0','false','true','false','(920) 496-8147','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','true','In-Store','false','(877) 363-4537','12:00AM - 12:00AM','12:00AM - 12:00AM','12:00AM - 12:00AM','12:00AM - 12:00AM','false','','true','false'],['299','West Bend','2180 S. Main Street','2180 S. Main Street','','/assets/temp/9_marker.png','West Bend, WI 53095','(262) 334-8500','Open 24 hrs a day, 364 days a year.','(262) 334-8510','9:00AM - 9:00PM','9:00AM - 7:00PM','10:00AM - 6:00PM','closed','West Bend','WI','53095','true','0','false','false','false','(262) 334-8547','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','7:00AM - 8:00PM','true','In-Store','false','(877) 363-4537','12:00AM - 12:00AM','12:00AM - 12:00AM','12:00AM - 12:00AM','12:00AM - 12:00AM','false','','true','false']]

Reply:
This above thing is example output data.

Reply:
The lat and long can be found further up on this page if you search for `setItem("LAT"`

Reply:
Ah, I see